easybuild/easyconfigs/m/Meep/Meep-1.4.3-intel-2020a.eb

```
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'Meep'
+version = '1.4.3'
+
+homepage = 'https://github.com/stevengj/meep'
+description = """Meep (or MEEP) is a free finite-difference time-domain (FDTD) simulation software package
+ developed at MIT to model electromagnetic systems."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+toolchainopts = {'usempi': True, 'opt': True, 'optarch': True, 'unroll': True, 'pic': True}
```

`optarch` is enabled by default, no need to enable it

```suggestion
toolchainopts = {'usempi': True, 'opt': True, 'unroll': True, 'pic': True}